src/ray/common/task/task_spec.h

```diff
@@ -159,7 +159,7 @@ static inline rpc::ObjectReference GetReferenceForActorDummyObject(
 class TaskSpecification : public MessageWrapper<rpc::TaskSpec> {
  public:
   /// Construct an empty task specification. This should not be used directly.
-  TaskSpecification() {}
+  TaskSpecification() { ComputeResources(); }
```

Comment:
this is for `task_manager_test`, where mock task spec doesn't currently compute resources, thus has nullpointers.